tests/integration/serve/test_streaming.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Maestro Streaming Test
+======================
+
+A single, self-contained test file that:
+1. Starts the Maestro workflow server automatically
+2. Tests the streaming functionality
+3. Verifies step-by-step responses
+4. Cleans up automatically
+
+Usage: python test_streaming.py
+"""
+
+import json
+import time
+import requests
+import subprocess
+import sys
+
+
+class StreamingTester:
```

Comment:
Would you change the class name to start with `Test` so that pytest picks this up?